app/models/client.rb

```diff
@@ -0,0 +1,3 @@
+class Client < ApplicationRecord
+  has_many :client_testimonial
```

Comment:
Will be client_testimonial or client_testimonials as it is has many
Have you tested this on local with multiple client testimonials

Reply:
Will be client_testimonial, it resembles to the controller file name created
Yes, tested on local

Reply:
![image](https://user-images.githubusercontent.com/64598190/188555441-fa4ea6be-6e5e-46f6-b1d7-f3d0570a5e93.png)